app/test_split_integer.py

```diff
@@ -2,20 +2,30 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert (
+            sum(split_integer(6, 2)) == 6
+    )
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    assert (
+            split_integer(25, 5) == [5, 5, 5, 5, 5]
+    )
 
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    assert (
+            split_integer(29, 1) == [29]
+    )
 
 
 def test_parts_should_be_sorted_when_they_are_not_equal() -> None:
-    pass
+    assert (
+            split_integer(13, 3) == [4, 4, 5]
+    )
 
 
 def test_should_add_zeros_when_value_is_less_than_number_of_parts() -> None:
-    pass
+    assert (
+            split_integer(4, 5) == [0, 1, 1, 1, 1]
+    )
```

Comment:
The test `test_should_add_zeros_when_value_is_less_than_number_of_parts` expects the function to return a list with zeros when the number of parts exceeds the value. However, the current implementation of `split_integer` does not handle this case, leading to unexpected results. Consider updating the function to handle cases where `number_of_parts` is greater than `value`.